src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,117 @@
-const Countdown = () => null;
+import React, { useState, useEffect } from 'react';
 
-export default Countdown;
+// Countdown starts at 2 minutes and 30 seconds (150000 milliseconds) and counts down to 0
+const Countdown: React.FC = () => {
+  const INITIAL_TIME = 150000; // 2 minutes and 30 seconds in milliseconds
+  const [time, setTime] = useState(INITIAL_TIME);
+  const [isRunning, setIsRunning] = useState(false);
+
+  useEffect(() => {
+    let intervalId: number;
+    if (isRunning && time > 0) {
+      intervalId = setInterval(() => {
```

Comment:
setInterval returns an interval that repeats every X ms. If you are going to clear the interval and recreate it each time, there's no point in using an interval but rather setTimeout, which runs just once.